tachyon/math/polynomials/multivariate/multilinear_dense_evaluations.h

```diff
@@ -186,7 +186,7 @@ class MultilinearDenseEvaluations {
         evaluations[b] = left + r * (right - left);
       }
     }
-    evaluations.resize(size_t{1} << (n - k));
+    evaluations.resize(size_t{1} << (n - k), F::Zero());
```

Comment:
It would be nice to be parallelized if the `(n-k)` is high.

Reply:
Removed. I think this `resize` always truncates the vector.